src/transformer_deploy/convert.py

```diff
@@ -337,61 +412,155 @@ def get_pytorch_infer(model: PreTrainedModel, cuda: bool, task: str):
         timings[engine_name] = time_buffer
         del engine, tensorrt_model, runtime  # delete all tensorrt objects
         gc.collect()
-        triton_conf.create_configs(
-            tokenizer=tokenizer, model_path=tensorrt_path, config=model_config, engine_type=EngineType.TensorRT
-        )
+        if commands.generative_model == "t5":
+            create_triton_configs(
+                tokenizer,
+                model_config,
+                pytorch_output,
+                EngineType.TensorRT,
+                commands.task,
+                commands.nb_instances,
+                input_names,
+                commands.output,
+                commands.device,
+            )
+        else:
+            triton_conf.create_configs(
+                tokenizer=tokenizer, model_path=tensorrt_path, config=model_config, engine_type=EngineType.TensorRT
+            )
 
     if "onnx" in commands.backend:
         num_attention_heads, hidden_size = get_model_size(path=commands.model)
         # create optimized onnx model and compare results
-        optimize_onnx(
-            onnx_path=onnx_model_path,
-            onnx_optim_model_path=onnx_optim_model_path,
-            fp16=run_on_cuda,
-            use_cuda=run_on_cuda,
-            num_attention_heads=num_attention_heads,
-            hidden_size=hidden_size,
-            architecture=model_config.model_type,
-        )
+        if commands.generative_model != "t5":
```

Comment:
Code duplicated. I think we can deduplicate ?

Reply:
done